config/locales/views/users/en.yml

```diff
@@ -24,6 +24,10 @@ en:
     not_successfully_added_message_3: "Student names are alphanumeric."
     not_successfully_added_message_4: "Each field is separated by a comma (no spaces)."
     not_successfully_added_message_5: "Each line has the required fields specified near the upload form."
+    toggle_invite_notifications: 'I would like to receive email notifications from MarkUs when I am invited to a group.'
```

Comment:
Remove the "I would like to" and "from MarkUs", so that this just says "Receive email notifications when I am invited to a group." Similarly for the one below.